### ISSUE TITLE
Import fails on Python 3

### ISSUE THREAD

**Issue by tylerdave** [Zack--/phonetics#1]
`import phonetics` fails on Python 3 (3.5 in my case) with an import error due to the relative import of utils:

```
      1 # -*- coding: utf-8 -*-
----> 2 import utils
      3 
      4 __ALL__ = [
      5     'metaphone',

ImportError: No module named 'utils'
```

**Comment by Zack--** [comment]
Fixed on master, will release a new version once #3 is fixed.

**Comment by ZackDibe** [comment]
Fix released in [`v1.0.5`](https://pypi.python.org/pypi/phonetics/1.0.5)
